notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb

```diff
@@ -0,0 +1,5688 @@
+{
```

Comment:
<p>remove Table of Contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>